docs/developer-guide/ksqldb-reference/aggregate-functions.md

```diff
@@ -221,6 +221,18 @@ Stream
 Return the minimum value for a given column and window.
 Rows that have `col1` set to null are ignored.
 
+## `STD_DEV_SAMP`
+
+Since: - 0.15.0
+
+```sql
+STD_DEV_SAMP(col1)

```

Comment:
`stddev_*` seems a common word used in other DBs. One _ less help write type less.

```suggestion
STDDEV_SAMP(col1)

```

Reply:
Good call, just made the change. Thanks @spena !